artmanflow/web/index.py

```diff
@@ -0,0 +1,45 @@
+# Copyright 2018 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from flask import Flask, Response, render_template, request, redirect, \
```

Comment:
As noted previously in a comment, I am really nervous about adding _an entire web application_ to artman. This is a non-trivial change with lots of implications beyond the scope of this PR.

If we are going to do this, we should carefully design it and have a plan for where it should go.

Reply:
No worries, I'm not adding it now to artman, but moving to a separate repo.